card.io/src/main/jni/nativeRecognizer.cpp

```diff
@@ -27,7 +27,7 @@ static int dmz_refcount = 0;
 static ScannerState scannerState;
 static bool detectOnly;
 static bool flipped;
-static bool lastFrameWasUsable;
```

Comment:
Yes, nowhere in the project it is was used or referenced. I just removed it because I stumbled upon it.